### PR TITLE
program1

### PR DESCRIPTION
**Pull request by rohit7747**

### CODE REVIEW

File: rohit.cpp [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	bzero(&client.sin_zero,0);
+	
+	if(connect(csock,(sockaddr *)&client,len)==-1)
+	{
+		perror("Connection not Established\n");
+		exit(-1);
+	}
+	
+	getwords();
+
```
Comment (rahulsinha2911) [inline rohit.cpp:40]: remove unnecessary spaces

Comment (rahulsinha2911) [reply rohit.cpp:40]: add this into your code `printf("first PR")`

Comment (rohit7747) [reply rohit.cpp:40]: ok

Comment (rohit7747) [reply rohit.cpp:40]: it's C++ prog. so sorry it would not work.

**Pull request merged**